_docs/tutorial/07-canary-release-with-istio.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Canary release with istio
+overview: Overview
+
+order: 07
+
+layout: docs
+type: markdown
+---
+{% include home.html %}\n# Canary Release with Istio
+In this learning module we will deploy a new version of the _reviews_ microservice again,
+this time with Istio enabled. We will release our new version to the `jason` user only (`jason` is our tester). It will allow the `jason` user to test the whole application end-to-end in production, with our new version.
```

Comment:
Maybe use a name other than jason, since it gets confusing relative to json IMO.

Reply:
@geeknoid I opened an issue https://github.com/istio/istio/issues/3945. Jason is used in Bookinfo sample rules, if we decide to change it we should rename it in istio/istio and istio/github.istio.io.